app/controllers/application_controller.rb

```diff
@@ -52,6 +52,7 @@ class ApplicationController < ActionController::Base
   include_concern 'TreeSupport'
   include_concern 'SysprepAnswerFile'
   include_concern 'ReportDownloads'
+  include_concern 'Show'
```

Comment:
I think application controller is already doing way too much .. adding this as a concern is definitely good, but I'd inlcude it only from controllers that actually need it, not application controller.